docs/introduction/vr-headsets-and-webxr-browsers.md

```diff
@@ -145,11 +145,11 @@ front-facing changes to A-Frame developers, involving mostly renaming of APIs.
 - Chrome (WebXR under origin trials)
 - Exokit (experimental early support)
 
-[webvrpolyfill]: https://github.com/googlevr/webvr-polyfill
```

Comment:
A-Frame still uses the WebVR polypill and no plans to migrate for now. We can revert this change